mlflow/store/artifact/databricks_artifact_repo.py

```diff
@@ -714,11 +752,19 @@ def get_creds_and_upload(staged_upload_chunk):
             for src_file_path, upload_future in inflight_uploads.items():
                 try:
                     upload_future.result()
+                    if pbar:
+                        pbar.update()
+                        pbar.refresh()
                 except Exception as e:
                     failed_uploads[src_file_path] = repr(e)
 
+        pbar = (
+            tqdm(total=len(staged_uploads), desc="Uploading artifacts") if _TQDM_AVAILABLE else None
+        )
```

Comment:
We only show the upload progress bar on databricks?

Reply:
For now yes, should we support everywhere?

Reply:
I think only supporting Databricks is fine.

Reply:
What about the download progress bar?